pkg/apis/settings/suite_test.go

```diff
@@ -80,7 +80,7 @@ var _ = Describe("Validation", func() {
 		_, err := (&settings.Settings{}).Inject(ctx, cm)
 		Expect(err).To(HaveOccurred())
 	})
-	It("should fail validation when batchIdleDuration is negative", func() {
+	It("should fail validation when batchIdleDuration is less then 1s", func() {
```

Comment:
We should actually validate that we are maintaining this minimum by testing a value that is just below that value. Neither of these tests ensure that. They both test negative values (which are good test cases, to be clear) but we also need other test cases for the min case.